proxy/http/remap/RemapPlugins.h

```diff
@@ -38,17 +38,12 @@
  * A class that represents a queue of plugins to run
  **/
 struct RemapPlugins : public Continuation {
-  RemapPlugins() : _cur(0), _rewriten(0) {}
+  RemapPlugins() = default;
```

Comment:
All of the members are declaration initialized, so the default constructor works fine. It does need to be declared so it's not suppressed by the other constructor.